Userland/Libraries/LibWeb/HTML/Scripting/ExceptionReporter.h

```diff
@@ -10,19 +10,14 @@
 
 namespace Web::HTML {
 
-enum class ErrorInPromise {
-    No,
-    Yes,
-};
```

Comment:
Please keep this, it's much clearer than an unnamed `true`/ `false` :)

Reply:
I decided to remove it because otherwise I would need to link LibWeb and use the `Web::HTML::ErrorInPromise` in LibJS just for the sake of this "boolean"-ish enum. Is it ok?

Reply:
I don't understand - while we are in LibWeb you can use this enum, and when calling into LibJS, use `value == ErrorInPromise::Yes` to get a boolean. It's not ideal, but better than bools everywhere.

Reply:
Didn't thought about that. Thanks for pointing out.